frontend/src/components/activePage/TableWorkspaceLogic/TableWorkspaceLogic.tsx

```diff
@@ -48,11 +49,29 @@ const TableWorkspaceLogic: FC<ITableWorkspaceLogicProps> = ({ ...props }) => {
     expandAll,
     setExpandAll,
     setCollapseAll,
+    advanced,
   } = props;
+
   const { tenantId, tenantNamespace } = user;
+  const [sortingData, setSortingData] = useState<
+    Array<{
+      sortingType: string;
+      sorting: number;
```

Comment:
```suggestion
      order: number;
```
Maybe order can be clearer?

Reply:
for a problem of conformity with the other elements reguarding this functionality i prefer to user "sorting"